src/MeetingsSDKAdapter.test.js

```diff
@@ -486,115 +486,117 @@ describe('Meetings SDK Adapter', () => {
+          disabledLocalAudio: null,
+          remoteAudio: {},
+        };
       });
-    });
 
-    test('throws errors if sdk meeting object is not defined', (done) => {
-      meetingsSDKAdapter.audioControl('inexistent').subscribe(
-        () => {},
-        (error) => {
-          expect(error.message).toBe('Could not find meeting with ID "inexistent"');
-          done();
-        },
-      );
-    });
-  });
+      test('does not call sdk muteAudio() if the meeting is inactive', async () => {
+        meetingsSDKAdapter.meetings[meetingID].remoteAudio = null;
+        await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
-  describe('handleLocalAudio()', () => {
-    beforeEach(() => {
-      meetingsSDKAdapter.meetings[meetingID] = {
-        ...meeting,
-        localAudio: {
-          stream: mockSDKMediaStreams.localAudio,
-        },
-        remoteAudio: {},
-      };
-    });
+        expect(mockSDKMeeting.muteAudio).not.toHaveBeenCalled();
+      });
 
-    test('skips muting audio if there is an inactive meeting', async () => {
-      meetingsSDKAdapter.meetings[meetingID].remoteAudio = null;
-      await meetingsSDKAdapter.handleLocalAudio(meetingID);
+      test('calls sdk muteAudio() if the meeting is active', async () => {
+        await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
-      expect(mockSDKMeeting.muteAudio).not.toHaveBeenCalled();
-    });
+        expect(mockSDKMeeting.muteAudio).toHaveBeenCalled();
+      });
 
-    test('skips unmuting audio if there is an inactive meeting', async () => {
-      meetingsSDKAdapter.meetings[meetingID].remoteAudio = null;
-      await meetingsSDKAdapter.handleLocalAudio(meetingID);
+      test('updates the meeting object to have audio muted', async () => {
+        await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
-      expect(mockSDKMeeting.unmuteAudio).not.toHaveBeenCalled();
-    });
+        expect(meetingsSDKAdapter.meetings[meetingID].localAudio.stream).toBeNull();
+      });
 
-    test('mutes audio if the the audio track is enabled', async () => {
-      await meetingsSDKAdapter.handleLocalAudio(meetingID);
+      test('emits a meeting updated event', async () => {
+        await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
-      expect(mockSDKMeeting.muteAudio).toHaveBeenCalled();
-    });
+        expect(mockSDKMeeting.emit).toHaveBeenCalledTimes(1);
+        expect(mockSDKMeeting.emit.mock.calls[0][0]).toBe('adapter:meeting:updated');
+        expect(mockSDKMeeting.emit.mock.calls[0][1]).toMatchObject({
+          localAudio: {stream: null},
+          disabledLocalAudio: mockSDKMediaStreams.localAudio,
+        });
+      });
 
-    test('localAudio.stream property should be null once the audio track is muted', async () => {
-      await meetingsSDKAdapter.handleLocalAudio(meetingID);
+      test('logs error if the sdk muteAudio() rejects with an error', async () => {
+        mockSDKMeeting.muteAudio = jest.fn(() => Promise.reject());
```

Comment:
in order to test the error being logged, you can reject with `'sdk error'` and add it to the expect() in place of `undefined`